contracts/lacchain.system/ricardian/lacchain.system.contracts.md

```diff
@@ -0,0 +1,399 @@
+title: Add Validator
+summary: 'Add to the network a Validator for the entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Validator for the entity.
+
+<h1 class="contract">addwriter</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Writer
+summary: 'Add to the network a Writer node for a Partner entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to Add to the network a Writer node for a Partner entity.
+
+<h1 class="contract">addboot</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Boot Node
+summary: 'Add a Boot Node to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to add a Boot Node to the network.
+
+<h1 class="contract">addobserver</h1>
+
+---
+spec_version: "0.2.0"
+title: Add Observer Node
+summary: 'Add a Observer Node to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to add a Observer Node to the network.
+
+<h1 class="contract">netaddgroup</h1>
+
+---
+spec_version: "0.2.0"
+title: Create a Group of Nodes
+summary: 'Create a Group of Nodes to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to add a Group of Nodes to the network.
+
+<h1 class="contract">rmnode</h1>
+
+---
+spec_version: "0.2.0"
+title: Remove a Node
+summary: 'Remove a Node from the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to remove a Node from the network.
+
+<h1 class="contract">netrmgroup</h1>
+
+---
+spec_version: "0.2.0"
+title: Remove a Group of Nodes
+summary: 'Remove a Group of Nodes from the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to remove a Group of Nodes from the network.
+
+<h1 class="contract">netsetgroup</h1>
+
+---
+spec_version: "0.2.0"
+title: Link a Node
+summary: 'Link a Node with a list of Groups to the network'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to link a Node with a list of Groups to the network.
+
+<h1 class="contract">setnodeinfo</h1>
+
+---
+spec_version: "0.2.0"
+title: Set Node Information
+summary: 'Set information to a Node'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to set information to a specific node.
+
+<h1 class="contract">setnodexinfo</h1>
+
+---
+spec_version: "0.2.0"
+title: Set Node Extended Information
+summary: 'Set extendend information to a Node'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to set extendend information to a specific node.
+
+<h1 class="contract">setentinfo</h1>
+
+---
+spec_version: "0.2.0"
+title: Set Entity Information
+summary: 'Set information to an Entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to set information to an entity.
+
+<h1 class="contract">setentxinfo</h1>
+
+---
+spec_version: "0.2.0"
+title: Set Extended Entity Information
+summary: 'Set extended information to an Entity'
+icon: @ICON_BASE_URL@/@ADMIN_ICON_URI@
+---
+
+Allows the user to set extendend information to an entity.
```

Comment:
please spell check:  extendend

I believe this action can only be performed by the permisioning committee
 please check required auth for this action

Reply:
No, It doesn't but `setnodeinfo` does

Reply:
Correction, functions makes a `get_self()` which before a I didn't know that eosio is the permissioning committee so then